client/dashboard/sites/overview/site-overview-fields.tsx

```diff
@@ -0,0 +1,101 @@
+import { useQuery } from '@tanstack/react-query';
+import { Link } from '@tanstack/react-router';
+import {
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	ExternalLink,
+} from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { siteAgencyBlogQuery } from '../../app/queries/site-agency';
+import { hasAtomicFeature } from '../../utils/site-features';
+import { isSelfHostedJetpackConnected } from '../../utils/site-types';
+import { getFormattedWordPressVersion } from '../../utils/wp-version';
+import { HostingFeatures } from '../features';
+import { PHPVersion } from '../site-fields';
+import type { Site } from '../../data/types';
+import './site-overview-fields.scss';
+
+const Field = ( { children, title }: { children: React.ReactNode; title?: React.ReactNode } ) => {
+	return (
+		<HStack
+			className="site-overview-field"
+			spacing={ 1 }
+			style={ { width: 'auto', flexShrink: 0 } }
+		>
+			{ title && <Text variant="muted">{ title }</Text> }
+			<div className="site-overview-field-children">{ children }</div>
+		</HStack>
+	);
+};
+
+const HostingProvider = ( { site }: { site: Site } ) => {
+	const { data: agencyBlog, isLoading: isLoadingAgencyBlog } = useQuery( {
+		...siteAgencyBlogQuery( site.ID ),
+		enabled: site.is_wpcom_atomic,
+	} );
+
+	if ( isLoadingAgencyBlog ) {
+		return null;
+	}
+
+	if ( agencyBlog ) {
+		return <Field>{ __( 'Managed through Automattic for Agencies' ) }</Field>;
```

Comment:
As per QOBjujZah0UlGDDdLKZhzi-fi-7281_115692 
```suggestion
		return <Field>{ __( 'Managed by Automattic for Agencies' ) }</Field>;
```

Reply:
Done by https://github.com/Automattic/wp-calypso/pull/104714/commits/eadecebe3a587c498f42df1673e4c13ccffa9bb8.